### PR TITLE
Improve code style of generated stream classes

### PR DESCRIPTION
**Pull request by zbynek**

### CODE REVIEW

File: examples/JavaCC/src/main/javacc/JavaCC.jj [hunk 1]
```diff
@@ -653,19 +657,22 @@ void node_descriptor_expression()
...
   }
 }
 
 
-/**********************************************
- * THE JAVA GRAMMAR SPECIFICATION STARTS HERE *
- **********************************************/
+/*--------------------------------------------
+ * THE JAVA GRAMMAR SPECIFICATION STARTS HERE
+ *--------------------------------------------*/
```
Comment (kaikalur) [inline examples/JavaCC/src/main/javacc/JavaCC.jj:675]: In javacc-8 we want to remove the java grammar completely out of the javacc grammar if possible.

Comment (zbynek) [reply examples/JavaCC/src/main/javacc/JavaCC.jj:675]: that sounds like agood improvement too, but in scope of this PR I only wanted to improve the code style of generated classes without changing their functionality.

**Comment by kaikalur** [comment]
Sounds good! Can you make sure some of the grammar tests run? Once you confirm we can merge it. Thanks!

**Review by kaikalur — APPROVED** [review]
Great! Thanks for enabling checks in CI

**Comment by zbynek** [comment]
@kaikalur anything else needed before merging this?

**Pull request merged**